src/redist/targets/GenerateLayout.targets

```diff
@@ -466,6 +466,13 @@
                       Overwrite="true" />
   </Target>
 
+  <Target Name="LayoutWorkloadUserLocalMarker"
+          Condition="'$(DotNetBuildFromSource)' == 'true'">
+    <WriteLinesToFile File="$(RedistLayoutPath)metadata/workloads/$(CliProductBandVersion)00/userlocal"
```

Comment:
This `00` looks like it may not be correct for patch releases?

Reply:
nvmd, it's correct.

Reply:
Yeah, that property is rather confusing.